app/controllers/concerns/blacklight/default_component_configuration.rb

```diff
@@ -64,39 +65,26 @@ def add_show_tools_partial name, opts = {}
         end
       end
 
-      ##
       # Add a tool to be displayed for each document in the search results.
-      # @param partial [String] the name of the document partial
-      # @param opts [Hash]
-      # @option opts [Symbol,Proc] :if render this action if the method identified by the symbol or the proc evaluates to true.
-      #                             The proc will receive the action configuration and the document or documents for the action.
-      # @option opts [Symbol,Proc] :unless render this action unless the method identified by the symbol or the proc evaluates to true
-      #                             The proc will receive the action configuration and the document or documents for the action.
-      def add_results_document_tool name, opts = {}
+      # @see #add_nav_action for params
```

Comment:
My $0.02 is to keep the duplication or use a macro.  If I were a developer browsing the online documentation (which is probably a lot more common than you might think), I would find this `@see` indirection confusing and not user-friendly (and would expect to see the method documented in-line).

Proliferating a pattern of connecting method documentation based on sharing a method signature feels potentially problematic to me.  It seems like potential downside of reducing this duplication outweighs the benefit that it is giving us.  Again, just my $0.02, hopefully it's not too controversial.